api/src/application/api/api-routes.js

```diff
@@ -7,7 +7,11 @@ module.exports = (userService, mixedValidation, rowNotExists, rowExists) => [
     method: 'POST',
     path: '/api/invite',
     handler: (request, reply) => {
-      reply(userService.inviteUser(request.payload));
```

Comment:
This line should work exactly the same as what you replaced it with. Why the change?